### PR TITLE
e2e: mutual contact request basic checks

### PR DESCRIPTION
**Pull request by pavloburykh**
Basic mutual contact requests cases.

### CODE REVIEW

File: test/appium/tests/medium/test_chats_m.py [hunk 1]
```diff
@@ -996,3 +996,102 @@ def test_block_user_from_1_1_chat_header_check_mentions_and_push_notification_se
...
+    @marks.testrail_id(702376)
+    def test_mutual_cr_send_and_accept_cr(self):
+        self.home_1.home_button.click()
+
+        self.home_1.just_fyi('Entering 1-1 chat and sending contact request')
+        self.chat_1 = self.home_1.add_contact(self.public_key_2, add_in_contacts=False)
+        self.chat_1.send_contact_request("Hi, please add me to contacts")
+
+        self.home_1.just_fyi('Check pending request is displayed, input field still disabled')
+        if not self.chat_1.element_by_text('Pending...').is_element_displayed():
```
Comment (churik) [inline test/appium/tests/medium/test_chats_m.py]: replace with translation-id "contact-request-pending"

File: test/appium/tests/medium/test_chats_m.py [hunk 2]
```diff
@@ -996,3 +996,102 @@ def test_block_user_from_1_1_chat_header_check_mentions_and_push_notification_se
...
+        chat_2.send_message(message_from_receiver)
+        if not self.chat_1.chat_element_by_text(message_from_receiver).is_element_displayed():
+            self.errors.append('Message from accepted user has not been received')
+
+        self.home_1.just_fyi('Verify chat input field has appeared after contact request has been accepted')
+        if not self.chat_1.chat_message_input.is_element_displayed():
+            self.drivers[0].fail('Chat input field has not appeared after contact request has been accepted')
+
+        self.home_1.just_fyi('Verify request sender can send messages to request acceptor after acceptance of cr')
+        self.chat_1.just_fyi("Clicking send button to clear input field from remained request message. Bug #13610")
```
Comment (churik) [inline test/appium/tests/medium/test_chats_m.py]: Better comment with #TODO : - they are reviewed periodically and easily can be found in code (built-in feature of PyCarm)

File: test/appium/tests/medium/test_chats_m.py [hunk 3]
```diff
@@ -996,3 +996,102 @@ def test_block_user_from_1_1_chat_header_check_mentions_and_push_notification_se
...
+    def test_mutual_cr_decline_contact_request(self):
+        [home.home_button.double_click() for home in (self.home_1, self.home_2)]
+        self.chat_1 = self.home_1.add_contact(self.public_key_2, add_in_contacts=False)
+        self.chat_1.send_contact_request('Contact request to decline')
+        self.home_2.handle_contact_request(self.default_username_1, accept=False)
+        self.home_2.home_button.click()
+        chat_2 = self.home_2.add_contact(self.public_key_1, add_in_contacts=False)
+
+        self.chat_1.just_fyi('Verify cannot sent messages to user who declined contact request')
+        if self.chat_1.chat_message_input.is_element_displayed():
```
Comment (churik) [inline test/appium/tests/medium/test_chats_m.py:1094]: Typo, `Verify cannot send messages to user who declined contact request`

**Review by churik — APPROVED** [review]

File: test/appium/tests/medium/test_chats_m.py (hunk 3 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_chats_m.py:1094]: Thanks @churik , fixed

File: test/appium/tests/medium/test_chats_m.py (hunk 2 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_chats_m.py]: added #TODO comment

File: test/appium/tests/medium/test_chats_m.py (hunk 1 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_chats_m.py]: @churik thanks, replaced

**Pull request merged**